pipelines/ConversionsForFakes.yaml

```diff
@@ -0,0 +1,54 @@
+# Add-on pipeline designed to be added to ApVerifyWithFakes or ApPipeWithFakes
+# for QA purposes.
+#
+# This pipeline depends on an external pipeline for the coaddName and fakesType
+# pipeline parameters.
+
+description: Type conversion tasks customized for AP pipeline
+imports:
+  - location: $AP_VERIFY_DIR/pipelines/Conversions.yaml
+    exclude:
+      # Fakes pipeline doesn't produce non-fakes DiaSourceTable.
+      - consolidateDiaSourceTable
+tasks:
+  # Conversion of fakes_src [afw.table] to fakes_source [Parquet]
+  writeSourceTableWithFakes:
+    class: lsst.pipe.tasks.postprocess.WriteSourceTableTask
+    config:
+      connections.catalogType: parameters.fakesType
```

Comment:
I'm sad that there's not a way to override all of the e.g. `catalogType` fields at once, but so be it.

Reply:
We actually don't want to do that, because we have non-fakes versions of these tasks in the assembled `ApPipeWithFakes`. They need to have different values for their templates.